src/App.tsx

```diff
@@ -3,9 +3,55 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
+import { getMovie } from './api';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [currentMovie, setCurrentMovie] = useState<null | Movie>(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const findMovieHandler = (userQuery: string) => {
+    setIsLoading(true);
+
+    getMovie(userQuery).then(data => {
```

Comment:
where do you `catching` your error?

Reply:
here     if ('Error' in data) {
      setCurrentMovie(null);
      setIsLoading(false);

      return;
    }

Reply:
check what the api function retruns for errors